### PR TITLE
DOCS: Update glossaries

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
## What

Document improvement. update glossaries

## Why ?

Same as #5538

## How ?

Update glossaries.

### CODE REVIEW

File: docs/source/glossaries.rst [hunk 1]
```diff
@@ -68,6 +71,7 @@ RDMA        Remote Direct Memory Access
 REQ         Request
 rkey        Remote KEY
 RMA         Remote Memory Access
+RNDV        Rendezvous
```
Comment (shamisp) [inline docs/source/glossaries.rst:74]: Rendezvous protocol

**Review by shamisp — CHANGES_REQUESTED** [review]

File: docs/source/glossaries.rst [hunk 2]
```diff
@@ -95,6 +100,7 @@ UD          Unreliable Datagram (InfiniBand)
 uGNI        user level generic network interface(Cray)
```
Comment (dmitrygx) [inline docs/source/glossaries.rst:100]: add whitespace between `interface` and `(Cray)`

File: docs/source/glossaries.rst [hunk 3]
```diff
@@ -95,6 +100,7 @@ UD          Unreliable Datagram (InfiniBand)
...
 UMR         User mode memory registration
 VPI         Virtual Protocol Interconnect
+WFE         Wait for event
 WIP-DNM     Work-in-Progress-Do-Not-Merge
 WQ          Work Queue(InfiniBand)
```
Comment (dmitrygx) [inline docs/source/glossaries.rst:105]: add whitespace between `Queue` and `(InfiniBand)`

pls, fix other places too
thank you

File: docs/source/glossaries.rst [hunk 4]
```diff
@@ -68,6 +71,7 @@ RDMA        Remote Direct Memory Access
 REQ         Request
 rkey        Remote KEY
 RMA         Remote Memory Access
+RNDV        Rendezvous
 RnR         Receiver Not Ready
 RoCE        RDMA over Converged Ethernet
 ROCm        Radeon Open Compute platform(AMD)
```
Comment (dmitrygx) [inline docs/source/glossaries.rst:77]: add whitespace between `platform` and `(AMD)`

File: docs/source/glossaries.rst (hunk 1 above)
Comment (hiroyuki-sato) [reply docs/source/glossaries.rst]: done

File: docs/source/glossaries.rst (hunk 2 above)
Comment (hiroyuki-sato) [reply docs/source/glossaries.rst]: done

File: docs/source/glossaries.rst (hunk 3 above)
Comment (hiroyuki-sato) [reply docs/source/glossaries.rst]: done. fix other places too.

File: docs/source/glossaries.rst (hunk 4 above)
Comment (hiroyuki-sato) [reply docs/source/glossaries.rst]: done

**Comment by hiroyuki-sato** [comment]
Hello, @shamisp and @dmitrygx.

Thank you for your comment. I fixed your comments. please take a look when you get a chance.

**Comment by shamisp** [comment]
Looks good, thanks !

**Review by shamisp — APPROVED** [review]

**Review by dmitrygx — APPROVED** [review]

**Comment by shamisp** [comment]
@hiroyuki-sato can you please squash . thanks !

**Comment by hiroyuki-sato** [comment]
@shamisp Thank you for your comment. rebased!

**Review by dmitrygx — APPROVED** [review]

**Pull request merged**